### PR TITLE
Added CheapShark API

### PR DESCRIPTION
**Pull request by ArmoredCavalry**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,6 +369,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Age of Empires II](https://age-of-empires-2-api.herokuapp.com) | Get information about Age of Empires II resources | No | Yes | Unknown |
 | [AmiiboAPI](http://www.amiiboapi.com/) | Amiibo Information | No | No | Yes |
 | [Battle.net](https://dev.battle.net/) | Blizzard Entertainment | `apiKey` | Yes | Unknown |
+| [CheapShark](https://www.cheapshark.com/api) | Steam/PC Game Prices and Deals | No | Yes | Yes |
```
Comment (53jk1) [inline README.md:372]: Maybe `https://www.cheapshark.com` instead of `https://www.cheapshark.com/api`?

**Review by 53jk1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ArmoredCavalry) [reply README.md:372]: @53jk1 - I believe the link should be to the API documentation, and not the homepage?

**Comment by yannbertrand** [comment]
Thanks!

I can't fix the conflict, could you please rebase your branch?

**Comment by ArmoredCavalry** [comment]
> I can't fix the conflict, could you please rebase your branch?

Looks like the conflict was resolved by your merge/commit?

**Comment by yannbertrand** [comment]
Oh. I just had to switch to another commit type. My bad.

**Comment by yannbertrand** [comment]
Thanks

**Comment by ArmoredCavalry** [comment]
Thank you!

**Pull request merged**